[2025-02-04T01:25:34.027+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-04T01:25:34.042+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nasa_dag.create_data manual__2025-02-04T01:25:33.107315+00:00 [queued]>
[2025-02-04T01:25:34.048+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nasa_dag.create_data manual__2025-02-04T01:25:33.107315+00:00 [queued]>
[2025-02-04T01:25:34.049+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-04T01:25:34.060+0000] {taskinstance.py:2888} INFO - Executing <Task(MongoDBOperator): create_data> on 2025-02-04 01:25:33.107315+00:00
[2025-02-04T01:25:34.064+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=148) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-04T01:25:34.066+0000] {standard_task_runner.py:72} INFO - Started process 149 to run task
[2025-02-04T01:25:34.066+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'nasa_dag', 'create_data', 'manual__2025-02-04T01:25:33.107315+00:00', '--job-id', '258', '--raw', '--subdir', 'DAGS_FOLDER/nasa_dag.py', '--cfg-path', '/tmp/tmplml62cuj']
[2025-02-04T01:25:34.068+0000] {standard_task_runner.py:105} INFO - Job 258: Subtask create_data
[2025-02-04T01:25:34.105+0000] {task_command.py:467} INFO - Running <TaskInstance: nasa_dag.create_data manual__2025-02-04T01:25:33.107315+00:00 [running]> on host fbf16bed15ba
[2025-02-04T01:25:34.165+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='nasa_dag' AIRFLOW_CTX_TASK_ID='create_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-04T01:25:33.107315+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-04T01:25:33.107315+00:00'
[2025-02-04T01:25:34.167+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-04T01:25:34.183+0000] {base.py:84} INFO - Retrieving connection 'mongo_default'
[2025-02-04T01:25:34.218+0000] {mongodb_file_operator.py:38} INFO - Collection etl_nasa already exists
[2025-02-04T01:25:34.228+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-04T01:25:34.228+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=nasa_dag, task_id=create_data, run_id=manual__2025-02-04T01:25:33.107315+00:00, execution_date=20250204T012533, start_date=20250204T012534, end_date=20250204T012534
[2025-02-04T01:25:34.280+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-04T01:25:34.303+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-04T01:25:34.306+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
